.github/workflows/pipeline.yml

```
@@ -0,0 +1,62 @@
+  pull_request:
+    types: [synchronize]
+
+jobs:
+  BackEnd-CI-CD:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)